### ISSUE TITLE
[Hunter] Animation bug - DPS drop

### ISSUE THREAD

**Issue by Ironjudgment** [SunstriderEmu/BugTracker#2357]
**Describe the bug**
Sometimes a hunter character gets stuck in animation and is unable to auto shoot or cast steady shot. The only thing I have noticed is that it tends to happen on target switch but NOT always. Sometimes it happens on single target. To exit such animation I had to either wait enough time or spam ESC several times to clear target and move a bit.

**To Reproduce**
I am not aware on how to reproduce this.

**Expected behavior**
A character should not be stuck in mid-animation.

**Screenshots/videos**
I have several from last night's Karazhan run. It's a long video so I'll just post links with time stamps on them.
https://youtu.be/nvWgg_5k4g0?t=323  -> Single trash mob
https://youtu.be/nvWgg_5k4g0?t=553 -> Animation stuck after casting multishot and changing target
https://youtu.be/nvWgg_5k4g0?t=755 - > Single target - Boss

And these were the first few minutes of the raid. There are planty more examples in the whole video, I'll look for more if requested.

**Additional context**
Only the standard 1:1 hunter macro was used to do rotation.

```
#showtooltip
/script UIErrorsFrame:Hide()
/castsequence reset=2.11 Steady Shot, !Auto Shot
/cast [exists,target=pettarget] Kill Command
/script UIErrorsFrame:Clear(); UIErrorsFrame:Show()

```

This macro didn't pose any issues while playing on Warmane/Sunwell/Atlantiss.

**Comment by Phandaroni** [comment]
Confirm, annoying problem!

**Comment by mayday2013** [comment]
Confirm, switching targets is pain. Unplayable in PvP.

**Comment by Highdps** [comment]
Animation randomly bugging, was testing haste effects doing this most offten but sometimes just normal rotation witchout anything getting bugged. 
If possible fix it it broke hunters and is unplayable.
Best regards

**Comment by Highdps** [comment]
Still game breaking, in open world are fine in raids it's bugging

**Comment by reinart1** [comment]
Can confirm, what ive also noticed is it happens a lot when feign deathing and being behind shaman totem, feels like getting LoS behind a totem after FD. Really weird, please look into it asap, cus its hurts DPS a lot.

**Comment by Ironjudgment** [comment]
Today at HKM it was a total clusterfuck. Four hunters constantly bugged, unable to even auto shoot. https://youtu.be/aafv8AppGCo EVERYSINGLE PULL

**Comment by reinart1** [comment]
they said after restart hunters will be fixed but im still experiencing the problem

**Comment by Ironjudgment** [comment]
Can confirm, still happening after fix.

**Comment by lukas155** [comment]
Still happening after the 'fix'. Seems to bug more severely in 25 mans. Sometimes can get locked out 
 of auto attacking for 10+ seconds while pressing absolutely nothing and still stuck in an animation.

**Comment by Godmatik** [comment]
Still happening, seems to happen to me after movement.

**Comment by Highdps** [comment]
Bug again appear, last 3 days are unplayable

**Comment by Jerrythemagicdog** [comment]
Fixed some time ago (8617ad2e922aa353b44bc0e4660a4f4587773526 and a94eaf7dbbae41e58653d09f7834742c49ac7f28 and 188f43740db69c8b080f9efa59ad13f74ef1f959)
